src/mono/wasi/Wasi.Build.Tests/ILStripTests.cs

```diff
@@ -0,0 +1,128 @@
+        if (expectILStripping)
+            Assert.True(Directory.Exists(strippedAssemblyDir), $"Could not find the stripped assemblies dir: {strippedAssemblyDir}");
+        else
+            Assert.False(Directory.Exists(strippedAssemblyDir), $"Expected {strippedAssemblyDir} to not exist");
+
+        string assemblyToExamine = "System.Private.CoreLib.dll";
+        string originalAssembly = Path.Combine(objBuildDir, origAssemblyDir, assemblyToExamine);
+        string strippedAssembly = Path.Combine(objBuildDir, strippedAssemblyDir, assemblyToExamine);
+        string includedAssembly = Path.Combine(appBundleFrameworkDir, assemblyToExamine);
+
+        Assert.True(File.Exists(originalAssembly), $"Expected {nameof(originalAssembly)} {originalAssembly} to exist");
+        if (!singleFileBundle)
+            Assert.True(File.Exists(includedAssembly), $"Expected {nameof(includedAssembly)} {includedAssembly} to exist");
+        if (expectILStripping)
+            Assert.True(File.Exists(strippedAssembly), $"Expected {nameof(strippedAssembly)} {strippedAssembly} to exist");
+        else
+            Assert.False(File.Exists(strippedAssembly), $"Expected {strippedAssembly} to not exist");
+
+        string compressedOriginalAssembly = Utils.GZipCompress(originalAssembly);
+        string? compressedIncludedAssembly = null;
+        FileInfo compressedOriginalAssembly_fi = new FileInfo(compressedOriginalAssembly);
+        FileInfo? compressedincludedAssembly_fi = null;
+
+        testOutput.WriteLine ($"compressedOriginalAssembly_fi: {compressedOriginalAssembly_fi.Length}, {compressedOriginalAssembly}");
+        if (!singleFileBundle)
+        {
+            compressedIncludedAssembly = Utils.GZipCompress(includedAssembly)!;
+            compressedincludedAssembly_fi = new FileInfo(compressedIncludedAssembly);
+            testOutput.WriteLine ($"compressedincludedAssembly_fi: {compressedincludedAssembly_fi.Length}, {compressedIncludedAssembly}");
+        }
+
+        if (expectILStripping)
+        {
+            string compressedStrippedAssembly = Utils.GZipCompress(strippedAssembly);
+            FileInfo compressedStrippedAssembly_fi = new FileInfo(compressedStrippedAssembly);
+            testOutput.WriteLine ($"compressedStrippedAssembly_fi: {compressedStrippedAssembly_fi.Length}, {compressedStrippedAssembly}");
+
+            // original > stripped assembly
+            Assert.True(compressedOriginalAssembly_fi.Length > compressedStrippedAssembly_fi.Length,
+                        $"Expected original assembly ({compressedOriginalAssembly}) size ({compressedOriginalAssembly_fi.Length}) " +
+                        $"to be bigger than the stripped assembly ({compressedStrippedAssembly}) size ({compressedStrippedAssembly_fi.Length})");
+
+            if (!singleFileBundle)
+            {
+                // original == included assembly
```

Comment:
The test is done correctly. However, the comment needs to be updated to `stripped == included assembly`.